### PR TITLE
Chakra

### PR DESCRIPTION
**Pull request by kristofferlarberg**
Tests the Chakra component library.

Possible shortcomings: no date/time picker, no search or multi select in select component.

This PR also uses react-datepicker.

Commented out `import 'antd/dist/antd.css';` to avoid pollution.

FireShot Capture 009 - UI library testbed - localhost

### CODE REVIEW

File: src/components/ui/AntDesign.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, {useState} from 'react';
 import { Avatar, Button, Cascader, Col, DatePicker, Input, Layout, Link, List, Row, Space, Tag, TextArea, TimePicker, Typography } from 'antd';
-import 'antd/dist/antd.css';
+/* import 'antd/dist/antd.css'; */
```
Comment (niklasva82) [inline src/components/ui/AntDesign.jsx]: Why did you comment this out? Is there some kind of incompatibility between the two different frameworks across components?

File: src/components/ui/Chakra.jsx [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+} from '@chakra-ui/react';
+import {
+	AddIcon,
+	DeleteIcon,
+	SearchIcon,
+	StarIcon,
+	SunIcon,
+} from '@chakra-ui/icons';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
```
Comment (niklasva82) [inline src/components/ui/Chakra.jsx:34]: Isn't the CSS for DatePicker imported as part of DatePicker?

**Review by niklasva82 — CHANGES_REQUESTED** [review]
A couple of things, but overall looks good.

File: src/components/ui/AntDesign.jsx (hunk 1 above)
Comment (kristofferlarberg) [reply src/components/ui/AntDesign.jsx]: Yes, it seems to apply on all other libraries if not commented out.

File: src/components/ui/Chakra.jsx (hunk 2 above)
Comment (kristofferlarberg) [reply src/components/ui/Chakra.jsx:34]: No, unfortunately not.

File: src/components/ui/AntDesign.jsx (hunk 1 above)
Comment (niklasva82) [reply src/components/ui/AntDesign.jsx]: Ah well that is problematic. It didn't seem to cause problems when merging grommet though.

Comment (kristofferlarberg) [reply src/components/ui/AntDesign.jsx]: The changes are minor, mainly margins on text, so if you want I can "uncomment" it out.

**Pull request merged**